app/models/dhs1171_pdf.rb

```diff
@@ -14,20 +14,25 @@ class Dhs1171Pdf
     "zip",
   ].freeze
 
-  SOURCE_PDF = "DHS_1171.pdf".freeze
+  PDF_DIRECTORY = "lib/pdfs".freeze
+  SOURCE_PDF = "#{PDF_DIRECTORY}/DHS_1171.pdf".freeze
+  COVERSHEET_PDF = "#{PDF_DIRECTORY}/michigan_snap_fax_cover_letter.pdf".freeze
 
-  def initialize(client_data)
+  def initialize(client_data:, output_filename: "test_output.pdf")
     @client_data = client_data
+    @working_filename = "tmp/working_#{output_filename}"
```

Comment:
I wonder if there is a better name for this than `working` -- maybe `filled_out_file_name` and then the output can be called `filled_out_file_with_coversheet_name` ...too verbose?